tests/busted/run_tests

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Just to be consistent with `check-all.sh`, and btw, `/bin/sh` sometimes points to other shells like `dash`, where BASH_SOURCE is not defined.